README.md

```diff
@@ -258,21 +258,590 @@ As part of the deployment, you'll need to set up [git integration](https://rasa.
+- In your AWS Console, go to **EC2 > Key Pairs**, and create a Key Pair with the name `findemo`, and download the file `findemo.pem` which contains the private SSH key. 
+  **Note that the name `findemo` is important, since it is used by the CI/CD pipeline when the cluster is created.**
+
+### Local AWS CLI
+
+Before the CI/CD pipeline can run, you will use the AWS CLI locally to create some items, so also install & configure the CLI locally.
+
+#### Install AWS CLI v2
+
+See the [installation instructions](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+
+#### Configure your AWS CLI
+
+```bash
+# AWS CLI version 2.1.26 or later
+aws --version
+
+# Configure AWS CLI
+aws configure
+AWS Access Key ID [None]: -----          # See above: IAM User API Keys
+AWS Secret Access Key [None]: -------    # See above: IAM User API Keys
+Default region name [None]: us-west-2    # The CI/CD pipeline uses us-west-2 
+Default output format [None]: 
+
+# Check your configuration
+aws configure list [--profile profile-name]
+
+# verify it works
+aws s3 ls
+```
+
+### ECR repository & S3 bucket
+
+The CI pipeline creates two artifacts:
+
+- An action server docker **image**, which is pushed to an **AWS ECR** repository.
+- A trained rasa **model**, which is copied to an **AWS S3** bucket
+
+Run these commands to create the storage for these artifacts. These commands run the AWS CLI and create an ECR repository and an S3 bucket:
+
+```bash
+# create ECR repository with name `financial-demo`
+make aws-ecr-create-repository
+
+# create S3 bucket with name `rasa-financial-demo`
+make aws-s3-create-bucket
+```
+
+
+
+## EKS production cluster
+
+If the production cluster is already set up, you can skip this section.
+
+This section describes the initial deployment of the financial-demo bot on an EKS production cluster. 
+
+This initial deployment is done manually. After that, the deployment is maintained & upgraded automatically by the CI/CD pipeline.
+
+### Preparation
+
+#### Install eksctl
+
+See the [installation instructions](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html)
+
+If you use Ubuntu, you can just issue the command:
+
+```bash
+make install-eksctl
+```
+
+#### Install kubectl
+
+See the [installation instructions](https://kubernetes.io/docs/tasks/tools/#kubectl)
+
+If you use Ubuntu, you can just issue the command:
+
+```bash
+make install-kubectl
+```
+
+#### Install helm
+
+See the [installation instructions](https://helm.sh/docs/intro/install/)
+
+If you use Ubuntu, you can just issue the command:
+
+```bash
+make install-helm
+```
+
+#### Install jp
+
+See the [installation instructions](https://github.com/jmespath/jp#installing)
+
+If you use Ubuntu, you can just issue the command:
+
+```bash
+make install-jp
+```
+
+#### Set environment variables
+
+There are many ways to set the required environment variables in your local environment. 
+
+One way is to create a file  `./secret/envs_export`, with this content:
```

Comment:
Can the GCR keys be set using the GCP CLI? Might be worth mentioning here as an alternative option.

Reply:
Wouldn't the GCP CLI be used only when you do things on a GCP VM?

This is just for defining the environments on your local computer.

Reply:
They can authenticate into the registry using the CLI as well: https://cloud.google.com/container-registry/docs/advanced-authentication#gcloud-helper. I think a link to their docs would be good enough if anyone is looking to authenticate a different way

Reply:
I added a link in the README.